tests/parser/functions/test_slice.py

```diff
@@ -83,12 +83,12 @@ def foo(inp: bytes <= 10, start: int128, len: int128) -> bytes <= 10:
     """
 
     c = get_contract_with_gas_estimation(test_slice4)
-    assert c.foo(b"badminton", 3, 3) == b"min"
-    assert c.foo(b"badminton", 0, 9) == b"badminton"
-    assert c.foo(b"badminton", 1, 8) == b"adminton"
-    assert c.foo(b"badminton", 1, 7) == b"adminto"
-    assert c.foo(b"badminton", 1, 0) == b""
-    assert c.foo(b"badminton", 9, 0) == b""
+    # assert c.foo(b"badminton", 3, 3) == b"min"
+    # assert c.foo(b"badminton", 0, 9) == b"badminton"
+    # assert c.foo(b"badminton", 1, 8) == b"adminton"
+    # assert c.foo(b"badminton", 1, 7) == b"adminto"
+    # assert c.foo(b"badminton", 1, 0) == b""
```

Comment:
Why are these commented out?

Reply:
Ha, thanks! That's not supposed to be there :see_no_evil: